src/nidcpower/metadata/functions_addon.py

```diff
@@ -49,6 +47,15 @@
     'DeleteAdvancedSequence':          { 'codegen_method': 'private',  },  # Advanced sequence private until #504 has a fix
 }
 
+functions_locking = {
```

Comment:
functions_locking seems odd... function_locking?

Reply:
Has to begin with 'functions_' for add_all to find it.